modules/Bio/EnsEMBL/Analysis/Hive/Config/HomologyRnaseq.pm

```diff
@@ -0,0 +1,295 @@
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+########################
+# db info
+########################
+    'genblast_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genblast_'.$self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_rnaseq_support_nr_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_gb_rnaseq_nr_'.$self->o('release_number'),
+      -host   => $self->o('genblast_rnaseq_support_db_server'),
+      -port   => $self->o('genblast_rnaseq_support_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'rnaseq_refine_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_refine_'.$self->o('release_number'),
+      -host   => $self->o('rnaseq_refine_db_server'),
+      -port   => $self->o('rnaseq_refine_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+  return [
+    # inheriting database and hive tables' creation
+    @{$self->SUPER::pipeline_create_commands},
+  ];
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    use_genome_flatfile  => $self->o('use_genome_flatfile'),
+    genome_file          => $self->o('faidx_genome_file'),
+  }
+}
+
+## See diagram for pipeline structure
+sub pipeline_analyses {
+  my ($self) = @_;
+  return [
+
+###############################################################################
+#
+# Homology Pipeline with RNAseq support
+#
+###############################################################################
+
+    {
+        -logic_name => 'create_genblast_rnaseq_support_db',
+        -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase',
+        -parameters => {
+          source_db   => $self->o('dna_db'),
+          target_db   => $self->o('genblast_rnaseq_support_db'),
+          create_type => 'clone',
+        },
+        -rc_name    => 'default',
+        -flow_into  => {
+          '1-A' => ['create_genblast_rnaseq_slice_ids'],
+          'A->1' => ['create_genblast_rnaseq_nr_db'],
```

Comment:
Could you please set the indentation for this block please? It should be easier to do it manually than with perltidy

Reply:
Done